core/services/keeper/registry_interface.go

```diff
@@ -175,6 +196,42 @@ func (rw *RegistryWrapper) GetActiveUpkeepIDs(opts *bind.CallOpts) ([]*big.Int,
 			activeUpkeepIDs = append(activeUpkeepIDs, activeUpkeepIDBatch...)
 		}
 
+		return activeUpkeepIDs, nil
+	case RegistryVersion_2_0:
```

Comment:
there was also registry1_3 added which has the same logic (you'll need to rebase). Post rebase lets try to consolidate the duplicate logic